coc/http.py

```diff
@@ -178,46 +170,46 @@ def __init__(self, client, loop, tokens=None, *, email, password, update_tokens=
 
     @staticmethod
     def create_cookies(response_dict, session):
-        return "session={};game-api-url={};game-api-token={}".format(
+        return "session={};game-api-url={};game-api-key={}".format(
```

Comment:
did you start doing regex change token to key? lol? this is how web stitches together cookies, so unless something changed, or there is a better way of doing this, pretty sure it was correct before

Reply:
I did a ctrlf to change token to key and that was after my tests oops I'll change that back thanks